### PR TITLE
Draft of a post on the decision to go virtual in 2021

### PR DESCRIPTION
**Pull request by funnelfiasco**
Fixes SeaGL/organization#56

### CODE REVIEW

File: _posts/2021-05-20-format-2021.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+categories: news
+---
+
+Like many other conferences, the SeaGL staff has given a lot of thought to the format.
+After much discussion and consideration of the general landscape, we have decided that **SeaGL 2021 will be a virtual event**.
+
+Last year's first-ever virtual SeaGL was a great success, and we're confident that we can deliver another delightful virtual experience this year!
+We loved how the remote format brought SeaGL to a wider audience, although we absolutely miss flocking together.
+
+Seagulls are not particularly known for clairvoyance, and we did't feel as though we could predict what the pandemic situation will look like in early November.
```
Comment (rachelkelly) [inline _posts/2021-05-20-format-2021.md:16]: spelling s/did't/didn't

File: _posts/2021-05-20-format-2021.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+We are encouraged by the progress being made on vaccinations, but we understand that many people will still be hesitant to attend in-person events.
+We're still holding open the idea of doing an in-person meetup for those of you in the Seattle area, but the conference proper will remain online in 2021.
+
+Join us on *the Internet*, Friday, November 5 and Saturday, November 6, 2021!
```
Comment (rachelkelly) [inline _posts/2021-05-20-format-2021.md:20]: :)

File: _posts/2021-05-20-format-2021.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+We will be announcing the Call for Participation soon.
```
Comment (rachelkelly) [inline _posts/2021-05-20-format-2021.md:22]: we tend to use the phrasing "call for proposas."  do we want to change it to the more inclusive sounding "call for participation"?  I kinda like it but we should be consistent regardless

**Review by rachelkelly — CHANGES_REQUESTED** [review]
one lil typo to fix, one consideration

File: _posts/2021-05-20-format-2021.md (hunk 3 above)
Comment (altsalt) [reply _posts/2021-05-20-format-2021.md:22]: I like that!

File: _posts/2021-05-20-format-2021.md (hunk 1 above)
Comment (funnelfiasco) [reply _posts/2021-05-20-format-2021.md:16]: Fixed

File: _posts/2021-05-20-format-2021.md (hunk 3 above)
Comment (funnelfiasco) [reply _posts/2021-05-20-format-2021.md:22]: I don't think I did this intentionally, but I like it. If we're agreed to use that phrasing everywhere, then I'll leave it.

**Comment by funnelfiasco** [comment]
Updated the date and ready to publish

**Comment by AndrewKvalheim** [comment]
> force-pushed the master branch from f15bc2a to 845f354

I think we lost 3e903e6 through f15bc2a.

Screenshot from 2021-06-08 13-27-48

**Comment by funnelfiasco** [comment]
WTFrenchfry? Okay, I'll re-add that file

**Comment by funnelfiasco** [comment]
Ah hell, looks like a bunch of commits got lost from master. Why did it do that, I only did a `--force-with-lease` on the branch I was writing the post in??? @AndrewKvalheim can you force push your copy to overwrite my tragic error and then I'll re-add my post

**Pull request merged**